src/sdk/bucketspace_fs.js

```diff
@@ -24,6 +25,7 @@ const { get_umasked_mode, isDirectory, validate_bucket_creation,
     create_config_file, delete_config_file, get_bucket_tmpdir_full_path, folder_delete } = require('../util/native_fs_utils');
 const NoobaaEvent = require('../manage_nsfs/manage_nsfs_events_utils').NoobaaEvent;
 const { anonymous_access_key } = require('./object_sdk');
+const manage_nsfs_cli_utils = require('../manage_nsfs/manage_nsfs_cli_utils');
```

Comment:
this dependency doesn't seem good, we don't want the bucketspace to depend on the cli code.

Reply:
I would remove it, the use is currently minor.